pkgs/build-support/dotnet/build-dotnet-module/default.nix

```diff
@@ -90,6 +93,10 @@ let
 
   inherit (callPackage ./hooks {
     inherit dotnet-sdk dotnet-test-sdk disabledTests nuget-source dotnet-runtime runtimeDeps buildType;
+    runtimeId =
+      if runtimeId != null
+      then [ runtimeId ]
```

Comment:
why is it put into an array?

Reply:
Good catch, that was unintentional.  I added an assert for the type in hooks/default.nix.